src/main/java/com/google/javascript/clutz/PlatformSymbols.java

```diff
@@ -63,7 +63,15 @@
    * Adding a symbol to this list also suppresses the duplicate externs.
    */
   public static final ImmutableSet<String> GLOBAL_SYMBOL_ALIASES =
-      ImmutableSet.of("Element", "Error", "Event", "EventTarget", "Object", "Date");
+      ImmutableSet.of(
+          "Date",
+          "Element",
+          "Error",
+          "Event",
+          "EventTarget",
+          "Float32Array",
+          "Float64Array",
+          "Object");
```

Comment:
Don;t you need to add them here too - https://github.com/angular/clutz/blob/master/src/resources/closure.lib.d.ts#L16

Reply:
Hah, what I had worked because it effectively made the code into implicit any :(